web.config

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     This configuration file is required if iisnode is used to run node processes behind
+     IIS or IIS Express.  For more information, visit:
+
+     https://github.com/tjanczuk/iisnode/blob/master/src/samples/configuration/web.config
+-->
+
+<configuration>
+    <system.webServer>
+        <!-- Visit http://blogs.msdn.com/b/windowsazure/archive/2013/11/14/introduction-to-websockets-on-windows-azure-web-sites.aspx for more information on WebSocket support -->
+        <webSocket enabled="false" />
+        <handlers>
+            <!-- Indicates that the server.js file is a node.js site to be handled by the iisnode module -->
+            <add name="iisnode" path="server.js" verb="*" modules="iisnode"/>
+        </handlers>
+        <rewrite>
+            <rules>
+                <!-- Do not interfere with requests for node-inspector debugging -->
+                <rule name="NodeInspector" patternSyntax="ECMAScript" stopProcessing="true">
+                    <match url="^server.js\/debug[\/]?" />
+                </rule>
+
+                <!-- First we consider whether the incoming URL matches a physical file in the /public folder -->
+                <rule name="StaticContent">
+                    <action type="Rewrite" url="public{PATH_INFO}"/>
```

Comment:
@MKaramath I was wondering whether we could use this to implement https://tools.hmcts.net/jira/browse/RDM-4090. We could just add a rewrite rule here that rewrites any request to /config to the homepage
@dwaynebailey you think this could work?

Reply:
A tmp fix maybe, but we'd want that I assume on non-prod envs I'd assume in which case we wouldn't be able to conditionally disable it in web.config.  Best to do it correctly in the app I think

Reply:
ok then. Perhaps @MKaramath best thing is to remove it from the `public` folder